usaspending_api/api_contracts/contracts/v2/reporting/agencies/publish_dates/publish_dates.md

```diff
@@ -0,0 +1,43 @@
+FORMAT: 1A
```

Comment:
The filename should match the endpoint route: `v2/reporting/agencies/agency_code/publish_dates.md`